src/dodal/devices/i15/focussing_mirror.py

```diff
@@ -0,0 +1,71 @@
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motor import Motor
+
+
+class FocusingMirrorBase(StandardReadable):
+    """Focusing Mirror with curve, ellip & pitch"""
+
+    def __init__(
+        self,
+        prefix: str,
+        name: str = "",
```

Comment:
Should: You can get all of this on one line (same with the other devices in this file), which will help with some of the verbosity here. If you remove the `,` after `name: str = ""` then ruff will auto reformat it on to the one line for you